snap/frontend/components/Button/Button.spec.jsx

```diff
@@ -0,0 +1,56 @@
+import Button from '.';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+describe('components/Button', () => {
+	it('renders Button', () => {
+		// Arrange:
```

Comment:
what you think about ``can render`` instead of ``renders Button`` since the main describe is ``Button``?  @Jaguar0625 might have a better suggestion.